src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center center;
+  }
+
+  &:not(:last-child) {
+    margin-right: 4px;
+  }
+
```

Comment:
You did a mistake. 
![image](https://user-images.githubusercontent.com/33294729/173564330-21c6f626-b59b-48d7-a3d6-137b43d415ff.png)

Reply:
wooow( thanks for paying attention!